AI-and-Analytics/End-to-end-Workloads/README.md

```diff
@@ -22,27 +22,9 @@ You can find more information at
 | Intel Extension for PyTorch (IPEX), Intel Neural Compressor (INC)           | [LanguageIdentification](LanguageIdentification) | Trains a model to perform language identification using the Hugging Face Speechbrain library and CommonVoice dataset, and optimized with IPEX and INC.
```

Comment:
Has IPEX became on official acronym for Intel Extension for PyTorch or it is for internal usage only? As I know, it's an internal acronym that should not be used publicly. Maybe I've missed something and it's officially approved? The same question for INC

Reply:
addressed it accordingly. remove acronym